apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -124,6 +169,43 @@ export default class WorkshopEnrollmentSchoolInfo extends React.Component {
           {this.props.accountRequiredForAttendance && (
             <td>{enrollment.user_id ? 'Yes' : 'No'}</td>
           )}
+          {this.props.workshopSubject === LOCAL_SUMMER && (
+            <td>
+              {enrollment.attendances} / {this.props.numSessions}
+            </td>
+          )}
+          {this.props.workshopSubject === LOCAL_SUMMER &&
+            this.state.pendingScholarshipUpdates.includes(enrollment.id) && (
+              <td>
+                <Spinner size="small" />
+              </td>
+            )}
+          {this.props.workshopSubject === LOCAL_SUMMER &&
```

Comment:
I wonder if this logic and the similar set below might benefit from a one-line comment summarizing what it's determining..